docs/develop/recipes/balance-conditional-transfers.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_position: 4
+---
+
+# Balance-Conditional Transfers
+
+In some use cases, you may want to execute a transfer if and only if an account has a certain
+balance.
+
+It would be unsafe to check an account's balance using the
+[`lookup_accounts`](../../reference/operations/lookup_accounts.md) and then perform the transfer,
+because these operations are not be atomic and the account's balance may change between the lookup
+and the transfer.
+
+You can atomically run a check against an account's balance before executing a transfer by using a
+control or temporary account and linked transfers.
+
+## Preconditions
+
+### 1. Target Account Must Have a Limited Balance
+
+The account for whom you want to do the balance check must have one of these flags set:
+
+- [`flags.debits_must_not_exceed_credits`](../../reference/accounts.md#flagsdebits_must_not_exceed_credits)
+  for accounts with [credit balances](../data-modeling.md#credit-balances)
+- [`flags.credits_must_not_exceed_debits`](../../reference/accounts.md#flagscredits_must_not_exceed_debits)
+  for accounts with [debit balances](../data-modeling.md#debit-balances)
+
+### 2. Create a Control Account
+
+There must also be a designated control or temporary account. As you can see below, this account
```

Comment:
Fwiw I like the term "control account" better... "temporary account" makes it sound like we are only using it once (which shouldn't be the case), instead of reusing it.